todo.py

```diff
@@ -32,6 +32,19 @@ def add_task_date(self, task_number, due_date):
             print(f'Task updated: {self.tasks[task_number - 1]}')
 
 
+    def update_task(self, task_number, updated_message):
+        """Change the description of a task"""
+        if task_number <= 0 or task_number > len(self.tasks):
+            print("Invalid task number!")
+        else:
+            task = self.tasks[task_number - 1]
+            if type(task) is str:
```

Comment:
Good job! I think it would be a good idea to notify the user if the task was updated successfully. I see that incase the user enters an invalid number it displays: "Invalid task number!" so if the task is updated successfully maybe we can display "Task successfully edited".

Reply:
Good idea, added.